src/config/check_element.c

```diff
@@ -16,12 +16,87 @@
 ** [Note] On success or on failure, close the file descriptors
 */
 
-void	check_texture()
+void	check_texture(char **element)
 {
+	struct stat	path_stat;
 
+	if (stat(element[1], &path_stat) != 0 || !S_ISREG(path_stat.st_mode))
+	{
+		ft_free_split(&element);
+		exit_error(EPATH);
+	}
+	if (ft_strncmp(element[1] + ft_strlen(element[1]) - 4, ".xpm", 4) != 0)
+	{
+		ft_free_split(&element);
+		exit_error(EEXP);
+	}
 }
 
-t_img	*load_img()
+t_img	*load_img(char **element, t_cub3d *world)
 {
-	
-}
\ No newline at end of file
+	int		height;
+	int		width;
+	t_img	*img;
+
+	img = mlx_xpm_file_to_image(world->graphics.mlx, \
+		element[1], &width, &height);
+	if (!img)
+	{
+		ft_free_split(&element);
+		exit_error(EIMG);
```

Comment:
I propose that the name of this error is `EXPM` and its content is `cub3d: Invalid xpm file`.